exclude_files.txt

```diff
@@ -1,2 +1,11 @@
 .git*
-*.py
```

Comment:
I think you still want to exclude `*.py` files. The compile process turns these into `.pyc` files, and their corresponding source files should no longer be needed.

Reply:
`exclude_fiules.txt` affects the compilation of the tarball on the host machine; it's not actually consumed by the Phantom `compile_app.pyc` script. So since we need all the Python source files in the tarball I think this line ought to remain deleted.

Reply:
well _that's_ confusing -- why is this file provided by Phantom if nothing internal to Phantom consumes it? Is it perhaps only relevant to _some_ of the options of `compile_app.pyc`?

Reply:
I think I actually just misinterpreted what Dan said in his presentation. Now I believe that the `exclude_files.txt` is only included in the app wizard boilerplate as a convenience to suggest to the user "if you want to exclude files from the app deployed to the Phantom server, use this file to omit those files during archive creation".

Reply:
It looks like that script has a `-g` flag to perhaps pass this file to?

```
[phantom@phantom ~]$ phenv python3 /opt/phantom/bin/compile_app.pyc -h
usage: compile_app.pyc [-h] (-s SINGLE_PYFILE | -i | -t | -c) [-a APP_DIR]
                       [-d] [-e] [-g IGNORE_FILE]

optional arguments:
  -h, --help            show this help message and exit
  -s SINGLE_PYFILE, --single_pyfile SINGLE_PYFILE
                        Compile a Single python file and exit
  -i, --install_app     Install app after compilation
  -t, --create_tarball  Only create the app tarball and exit, compilation or
                        installation is not carried out
  -c, --compile_app     Compile the app and exit
  -a APP_DIR, --app_dir APP_DIR
                        app directory
  -d, --exclude_flake   Dont run flake
  -e, --continue_on_error
                        Stop on error
  -g IGNORE_FILE, --ignore_file IGNORE_FILE
                        files that contains the list of files to ignore, by
                        default it is .compile_app.ignore
```

Reply:
Alan and I discussed this. For the `deploy` command, we decided to perform the exclusion at tar creation time. If in the future we extend this script to add a command just for compiling the app, we will use the `-g` argument of the `compile_app.pyc` script against the same `exclude_files.txt` file.